zebrad/tests/common/get_block_template_rpcs/get_block_template.rs

```diff
@@ -90,23 +102,29 @@ pub(crate) async fn run() -> Result<()> {
 
     assert!(is_response_success);
 
+    tokio::time::sleep(EXPECTED_MEMPOOL_TRANSACTION_TIME).await;
```

Comment:
Optional: Should we run our first iteration with an empty mempool, for better test coverage?

Reply:
Yes! I wasn't entirely sure why that was there before so I didn't want to change it, but I don't like it and will get rid of it if that's okay.